tests/upgrade/test_upgrade_from.sh

```diff
@@ -35,75 +35,67 @@ LOCAL_FORTIO_LOG=/tmp/fortio_local.log
 POD_FORTIO_LOG=/tmp/fortio_pod.log
 
 # Make sure to change templates/*.yaml with the correct address if this changes.
-TEST_NAMESPACE=test
+TEST_NAMESPACE="test"
```

Comment:
Quotes are not needed. I saw that a lot of changes simply add double quotes, which are not necessary at all.

Reply:
ShellCheck complains about lack of quotes

Reply:
surprised that it'd complain about lacking of quotes in the above statement.